docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
+
+```bash
+wget https://github.com/canonical/pebble/releases/download/v1.11.0/pebble_v1.11.0_linux_amd64.tar.gz
+tar zxvf pebble_v1.11.0_linux_amd64.tar.gz
+sudo mv pebble /usr/local/bin/ # make sure it's in your $PATH
+```
+
+## Verify Pebble installation
+
+Once the installation is complete, verify that `pebble` has been installed correctly by running:
+
+```bash
+pebble
+```
+
+This should produce output similar to the following:
+
+```bash
+Pebble lets you control services and perform management actions on
+the system that is running them.
+
+Usage: pebble <command> [<options>...]
+
+...
+```
+
+## Configure Pebble
+
+Now that Pebble has been installed, we can set up a basic configuration.
+
+First, let's create a directory for Pebble configuration:
+
+```bash
+mkdir -p ~/PEBBLE/layers
+export PEBBLE=$HOME/PEBBLE
+echo "export PEBBLE=$HOME/PEBBLE" >> ~/.bashrc # add $PEBBLE to your bashrc
+```
+
+Next, create a layer by running:
+
+```bash
+echo """\
+summary: Simple layer
+
+description: |
+    A simple layer.
+
+services:
+    http-server:
+        override: replace
+        summary: demo http server
+        command: python3 -m http.server 8080
+        startup: enabled
+""" > $PEBBLE/layers/001-http-server.yaml
+```
+
+This creates a simple layer containing only one service (which runs a basic HTTP server using Python's `http` module, listening on port 8080).
+
+## Start the Pebble daemon
+
+Now we are ready to run the Pebble daemon.
+
+Pebble is invoked using `pebble <command>`. (To get more information, run `pebble -h`.)
+
+To start the daemon, run:
+
+```bash
+pebble run
+```
+
+This starts the Pebble daemon itself, as well as all the services that are marked as `startup: enabled` (in our simple layer created above, the `http-server` service is marked as `startup: enabled`). You should get some output similar to the following:
+
+```bash
+2024-06-02T11:30:02.925Z [pebble] Started daemon.
+2024-06-02T11:30:02.936Z [pebble] POST /v1/services 10.751704ms 202
+2024-06-02T11:30:02.936Z [pebble] Started default services with change 77.
+2024-06-02T11:30:02.942Z [pebble] Service "http-server" starting: python3 -m http.server 8080
+...
+```
+
+As you can see from the log, our HTTP server is started too, which can be verified by running `curl localhost:8080` in another terminal tab.
+
+> Note: To exit the Pebble daemon, press Ctrl-C (which sends an "interrupt" signal to the process).
+
+## View, start and stop services
+
+While the Pebble daemon is running, you can view the status of services by running `pebble services`. Open another terminal tab, and run:
+
+```bash
+pebble services
+```
+
+You should see output similar to the following:
+
+```bash
+Service      Startup  Current  Since
+http-server  enabled  active   today at 11:30 UTC
+```
+
+Use `pebble stop <service1> <service2> ...` to stop one or more services. You can stop the running `http-server` service by running:
+
+```bash
+pebble stop http-server
+```
+
```

Comment:
```suggestion
```{tip}
To stop one or more running services, run `pebble stop <service1> <service2>`.

```

This reference can be moved to the end since it is not needed as part of the current learning journey.

Reply:
Can't be rendered by markdown.